### ISSUE TITLE
How to change to multiSort Icons ?

### ISSUE THREAD

**Issue by snoopy72** [wenzhixin/bootstrap-table#5325]
**Description**  
Running the current version 1.18.0 i have the following problem:

i would like to change the icon of the multi-sort button in the upper right.

normally (for the other buttons) i am able to do it by
```js
iconsPrefix: "",
icons: {
     refresh: "las la-sync",
```

Due to logic (and some thin documentation) i should be:
```js
     multipleSort: "las la-sort",
OR
     multi-sort: "las la-sort",
OR
     multiSort: "las la-sort",
OR
     multisort: "las la-sort",
```

But i found no way to change the glyphicon with my icon for this extension.
Also trying with "Icons", "Icon" changed nothing.

I was able to change the popup info with following code:
```js
formatMultipleSort: function formatAutoRefresh() {
     return "Eigener Text";
},
```

**Comment by snoopy72** [comment]
Example:
https://live.bootstrap-table.com/code/snoopy72/4872

btw: in my example i was able to change the auto-update icon (ticket #5323)

**Comment by snoopy72** [comment]
Figured out the following phenomenon:

The following code seems to work,
```js
iconsPrefix: "las",
icons: {
      	refresh: "la-sync",
        autoRefresh: "la-clock",
        multipleSort: "la-sort",
},
```

BUT the this one not:
```js
iconsPrefix: "",
icons: {
      	refresh: "las la-sync",
        autoRefresh: "las la-clock",
        multipleSort: "las la-sort",
},
```

hm. Problem in my case: I cannot use the prefix, because some buttons are in font-awesome and some in line-awesome by customer wishes, so no change to change it. For all other buttons this is working quite well - but not for multiple sort.

**Comment by UtechtDustin** [comment]
Currently not possible to customize the icon, its fixed in #5329
